test/server/config_validation/xds_fuzz.cc

```diff
@@ -189,12 +189,16 @@ void XdsFuzzTest::replay() {
   // URL so just don't check them until a listener is added
   bool sent_listener = false;
 
+  uint32_t added = 0;
```

Comment:
Are these just for future accounting?

Reply:
I used them below on lines 212 and 215 to wait for the updates to come into effect

Reply:
Ahhh sorry I missed the expectation compare. thanks!